### PR TITLE
[sysdig, sysdig-deploy, node-analyzer] Feat/chart bumps

### PR DESCRIPTION
**Pull request by cgeers**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped for the respective charts
- [x] Changelog updated for the charts with version updates.
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Comment by cgeers** [comment]
I believe this is ready.

File: charts/node-analyzer/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 ## Change Log
 
 This file documents all notable changes to Sysdig Node Analyzer Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+## 1.7.31
+### Minor changes:
+* BenchmarkRunner
+    * bump to version 1.1.0.2
+        * bump linux bench to 1.1.0.1
```
Comment (aroberts87) [inline charts/node-analyzer/CHANGELOG.md]: nit: isn't this an unnecessary level of indentation?

Comment (aroberts87) [inline charts/node-analyzer/CHANGELOG.md]: from this PR i cannot see anything bumped _to_ version `1.1.0.1`, only from `1.1.0.1` to `1.1.0.2`. does this entry, and the one in the other changelog, need to be adjusted?

File: charts/sysdig-deploy/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## Change Log
 
+## v1.3.40
+### Minor changes:
+* node-analyzer bumping version to 1.7.31
+
+
```
Comment (yashgiri) [inline charts/sysdig-deploy/CHANGELOG.md]: Also a small nit can this extra line space be removed

File: charts/node-analyzer/CHANGELOG.md (hunk 1 above)
Comment (yashgiri) [reply charts/node-analyzer/CHANGELOG.md]: yup could use better wording.

Comment (cgeers) [reply charts/node-analyzer/CHANGELOG.md]: it's a nested dependency. I'll just remove it.

Comment (cgeers) [reply charts/node-analyzer/CHANGELOG.md]: the versions, as strange as they may seems, are correct. I'll update to remove the versions that reference embedded components.

**Comment by cgeers** [comment]
thanks for the feedback. responded and updated.

**Review by lilx1ao — APPROVED** [review]

**Review by aroberts87 — APPROVED** [review]

**Review by yashgiri — APPROVED** [review]

**Pull request merged**